lib/wanda/checks_selection.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Wanda.ChecksSelection do
+  @moduledoc """
+  Entry point to interact with a targeted checks selection.
+  """
+
+  alias Wanda.Catalog
+  alias Wanda.Catalog.{CheckCustomization, SelectableCheck, Value}
+  alias Wanda.ChecksCustomizations
+
+  @spec selectable_checks(group_id :: String.t(), env :: map()) :: [SelectableCheck.t()]
+  def selectable_checks(group_id, env) do
+    available_customizations = ChecksCustomizations.get_customizations(group_id)
+
+    env
+    |> Catalog.get_catalog()
+    |> Enum.map(
+      &%SelectableCheck{
```

Comment:
This is a bit harder to read,  what about extracting the check building  to a function ?

Reply:
Done 2d9ef9d8b980b22d50ba3bf3c298caf373453ea7